docs/rooms/fireRoom.md

```diff
@@ -0,0 +1,59 @@
+# Fire Room
+
+A room with three different paths.
+One path leads to a person, the second path leads to a item and the third path leads nowhere.
+Each path is on fire but the player is given two buckets of water.
+The goal of this room is to save the person.
+
+## PlayerStates
+
+- **AtEntrance**: You are in the main room with X buckets of water.
+  There is a fire to your left/right/in front of you.
+  You have/haven't saved the person and you have/haven't retrieved the item.
+- **Douse**: The water from the bucket puts out the fire.
+- **Nothing**: There is no water left. You did nothing.
+- **SavedPerson**: Congratulations!!! You saved the person!!!
+- **ItemRetrieved**: Congratulations!!! You got a cool item!!!
+- **EmptyPath**: There is nothing here. Better luck next time.
+- **Flameless**: You already put out the fire here...
+
+## RoomState
+
+- **waterBuckets**: The number of buckets that still have water in them.
+- **fireForward**: Boolean for if there is a fire in front of you.
+- **fireLeft**: Boolean for if there is a fire to the left of you.
+- **fireRight**: Boolean for if there is a fire to the right of you.
+- **personSaved**: Boolean for if the person was saved.
+- **itemRetrieved**: Boolean for if the item was found.
+
+## Commands
+
+- **leave**: You leave through the only available door.
+  X of the fires were put out and there was X number of water buckets left.
+  The person was/wasn't saved and you did/didn't find the item.
+- **throwBucket**: You throw the bucket of water
+  - **forward (default)**: in front of you.
+  - **left**: to the left of you
+  - **right**: to the right of you.
+
+## Diagram
+
+```mermaid
+stateDiagram-v2
+  [*] --> AtEntrance
+
+  AtEntrance --> Douse: throwBucket (if there is water left)
+  AtEntrance --> Nothing: throwBucket (if there is no water left)
+  AtEntrance --> [*]: leave
+
+  Douse --> EmptyPath
```

Comment:
Double checking from the call... Should this just be `Douse --> AtEntrance` and get rid of all of the extra `PlayerStates`s? (since all of the info needed is outputted by `AtEntrance`)

Reply:
I'm not sure why "Douse" is a state. That sounds like it can be part of the command description for "throw-bucket". How come these relationships don't have an associated command?

Reply:
I also don't think you need any of these additional states like "Douse", "EmptyPath", "Flameless", "Nothing" etc

Reply:
Yep... This was way more complicated then it needs to be.